src/Altinn.Platform/Altinn.Platform.Storage/Storage/Controllers/ApplicationsController.cs

```diff
@@ -39,6 +40,7 @@ public ApplicationsController(IApplicationRepository repository, ILogger<Applica
         /// </summary>
         /// <param name="org">application owner id</param>
         /// <returns>list of all applications for a given owner</returns>
+        [Authorize(Policy = "InstanceRead")]
```

Comment:
Should we use a constant value instead of a string?
In case of a policy name change or something similar?
Same for all [Authorize(Policy = "...")]

Reply:
yeah